app/delivery/send_to_providers.py

```diff
@@ -42,9 +40,7 @@ def send_sms_to_provider(notification):
 
     if notification.status == 'created':
         provider = provider_to_use(SMS_TYPE, notification.id, notification.international)
-        current_app.logger.debug(
-            "Starting sending SMS {} to provider at {}".format(notification.id, datetime.utcnow())
-        )
```

Comment:
did we decide we don't need those logs anymore? Or are they sent from elsewhere?

Reply:
Debug statements don't show up in prod logs. Other log messages in the journey give us this information.

Reply:
Ahh, ok, makes sense 👍